src/core/models/answersconfig.js

```diff
@@ -0,0 +1,216 @@
+/** @module AnswersConfig */
+
+import { LOCALE, PRODUCTION, QUERY_SOURCE, SANDBOX } from '../constants';
+import SearchConfig from '../models/searchconfig';
+import VerticalPagesConfig from '../models/verticalpagesconfig';
+
+const sandboxPrefix = `${SANDBOX}-`;
+const DEFAULTS = {
+  environment: PRODUCTION,
+  onReady: function () {},
+  useTemplates: true,
+  locale: LOCALE,
+  experienceVersion: 'PRODUCTION',
+  debug: false,
+  sessionTrackingEnabled: true,
+  onStateChange: function () {},
+  onVerticalSearch: function () {},
+  onUniversalSearch: function () {},
+  disableCssVariablesPonyfill: false,
+  querySource: QUERY_SOURCE
+};
+
+export default class AnswersConfig {
+  constructor (config) {
+    const parsedConfig = this.parseConfig(config);
+
+    /**
+     * The internal Yext environment
+     * @type {string}
+     * @private
+     */
+    this._environment = parsedConfig.environment;
+
+    /**
+     * Required, the Yext Answers API Key
+     * @private
+     * @type {string}
+     */
+    this._apiKey = parsedConfig.apiKey;
```

Comment:
Can I set these as non-private and still have the getters for these?

Reply:
No, you cannot have a property hold a value and also have a getter bound to it. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#description